src/AppCockpit.tsx

```diff
@@ -0,0 +1,38 @@
+import '@patternfly/react-core/dist/styles/base.css';
+import '@patternfly/patternfly/patternfly-addons.css';
+
+import React from 'react';
+
+import NotificationsPortal from '@redhat-cloud-services/frontend-components-notifications/NotificationPortal';
+import { createRoot } from 'react-dom/client';
+import { Provider } from 'react-redux';
+import { HashRouter } from 'react-router-dom';
+
+import { Router } from './Router';
+import { store } from './store';
+
+const Application = () => {
+  return (
+    <React.Fragment>
+      <NotificationsPortal />
+      <HashRouter>
+        <Router />
+      </HashRouter>
+    </React.Fragment>
+  );
+};
+const ImageBuilder = () => (
+  <Provider store={store}>
+    <Application />
+  </Provider>
+);
+
+const main = async () => {
```

Comment:
This seems very similar to what we do in https://github.com/osbuild/image-builder-frontend/blob/main/src/bootstrap.tsx, could we re-use that code?

Reply:
Okay, I think it makes sense to have separate entry points!